sdk/jobs/automl-standalone-jobs/automl-image-classification-multilabel-task-fridge-items/automl-image-classification-multilabel-task-fridge-items.ipynb

```diff
@@ -204,6 +286,44 @@
+    "- `experiment_name` - The name of the experiment. An experiment is like a folder with multiple runs in Azure ML Workspace that should be related to the same logical machine learning experiment.\n",
+    "- `name` - The name of the Job/Run. This is an optional property. If not specified, a random name will be generated.\n",
+    "- `target_column_name` - The name of the column to target for predictions. It must always be specified. This parameter is applicable to 'training_data', 'validation_data' and 'test_data'.\n",
+    "- `primary_metric` - The metric that AutoML will optimize for model selection.\n",
+    "- `training_data` - The data to be used for training. It should contain both training feature columns and a target column. Optionally, this data can be split for segregating a validation or test dataset. \n",
+    "You can use a registered MLTable in the workspace using the format '<mltable_name>:<version>' OR you can use a local file or folder as a MLTable. For e.g Input(mltable='my_mltable:1') OR Input(mltable=MLTable(local_path=\"./data\"))\n",
+    "The parameter 'training_data' must always be provided.\n",
+    "\n",
+    "\n",
+    "### set_limits() parameters:\n",
+    "This is an optional configuration method to configure limits parameters such as timeouts.     \n",
+    "    \n",
+    "- timeout_minutes - Maximum amount of time in minutes that the whole AutoML job can take before the job terminates. This timeout includes setup, featurization and training runs but does not include the ensembling and model explainability runs at the end of the process since those actions need to happen once all the trials (children jobs) are done. If not specified, the default job's total timeout is 6 days (8,640 minutes). To specify a timeout less than or equal to 1 hour (60 minutes), make sure your dataset's size is not greater than 10,000,000 (rows times column) or an error results.\n",
+    "\n",
+    "- trial_timeout_minutes - Maximum time in minutes that each trial (child job) can run for before it terminates. If not specified, a value of 1 month or 43200 minutes is used.\n",
+    "    \n",
+    "- max_trials - The maximum number of trials/runs each with a different combination of algorithm and hyperparameters to try during an AutoML job. If not specified, the default is 1000 trials. If using 'enable_early_termination' the number of trials used can be smaller.\n",
+    "    \n",
+    "- max_concurrent_trials - Represents the maximum number of trials (children jobs) that would be executed in parallel. It's a good practice to match this number with the number of nodes your cluster.\n",
+    "    \n",
+    "- enable_early_termination - Whether to enable early termination if the score is not improving in the short term. \n",
+    "    "
+   ]
+  },
   {
    "cell_type": "code",
```

Comment:
It might be better to split this cell into multiple cells, say one for image_classification_multilabel() creation, one for set_limits() and one for search_space and one for sweep_settings. 
Since you are already planning to add the text about hyperparameter sweeping from v1 notebooks, you can add it at the corresponding cell.